### PR TITLE
docs(plugin): Add a README for using and writing plugins

### PR DESCRIPTION
**Pull request by juliemr**

### CODE REVIEW

**Comment by juliemr** [comment]
@sjelin can you review?

File: plugins/README.md [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+ * updated (if applicable).
+ *
+ * @param {Object} config The plugin configuration object.
+ *
+ * @return Return values are ignored.
+ */
+exports.postResults = function(config) {};
+```
+
+The protractor results object follows the following format:
```
Comment (sjelin) [inline plugins/README.md]: Maybe we should just link to the description here: https://github.com/angular/protractor/tree/master/lib/frameworks#requirements ?

**Comment by sjelin** [comment]
LGTM

File: plugins/README.md (hunk 1 above)
Comment (juliemr) [inline plugins/README.md]: Agreed that we should have one source of truth for this documentation. Done.

**Comment by juliemr** [comment]
Merged as 40453864d29fbb1bd22bc8bb201462a589119f57

**Pull request closed without merging**